### DISCUSSION TITLE
MIMIC icd codes

### DISCUSSION THREAD

**Discussion by msoren3214** [MIT-LCP/mimic-code#1486] [MIMIC-General] [answered]
Why are MIMIC icd diagnosis codes different from those presented using the National Center for Health Statistics ICD-10-CM Browser tool.  MIMIC-IV codes are otherwise the same but without decimal points. MIMIC-ED codes are totally different.

**Comment by alistairewj** [comment]
In what way are the MIMIC-IV-ED codes totally different? Can you provide some examples?

**Comment by msoren3214** [comment, marked as answer]
I realize now, the MIMIC-IV-ED codes are the same if decimal points are inserted.  Thanks.
